language/de/cli.php

```diff
@@ -120,12 +120,14 @@
 	'CLI_EXTENSION_ENABLE_FAILURE'		=> 'Konnte Erweiterung %s nicht aktivieren',
 	'CLI_EXTENSION_ENABLE_SUCCESS'		=> 'Erweiterung %s erfolgreich aktiviert',
 	'CLI_EXTENSION_ENABLED'				=> 'Erweiterung %s ist bereits aktiviert',
+	'CLI_EXTENSION_NOT_EXIST'			=> 'Erweiterung %s existiert nicht',
```

Comment:
Sollte es nicht heißen Die Erweiterung %s existiert nicht?
Wenn es so bleiben sollte, sollte der Satz Die Erweiterung %s konnte nicht aktualisiert werden angepasst werden.
Denn dann ist alles auch einheitlich.

Reply:
Sieht das noch jemand so?

Reply:
Einheitlich sollten wir auf jeden Fall werden. Aktuell ist nur Zeile 127 nicht einheitlich.

Da es immer nur um eine Erweiterung geht und das soweit ich weiß auch einzeln ausgegeben wird spricht aber auch in meinen Augen nichts dagegen, das "Die" hinzuzufügen. Dann aber gleich in Zeile 120, 121, 122, 126 und auch 130 (inklusive ggf. fehlender weiterer Worte). Andersrum gehört das "Die" in Zeile 127 raus, falls wir es weglassen.

Ob mit oder ohne "Die" ist mir relativ egal, ich bin aber eigentlich eher für kurze CLI Meldungen -> Ohne "Die".

Reply:
Habe es mal "ohne DIE" umgesetzt.